app/views/cars/edit.html.erb

```diff
@@ -29,22 +29,9 @@
         <%#= link_to(cl_image_tag(image.key, height: 200, width: 200, gravity: :auto, crop: :thumb, sign_url: true, fetch_format: :auto), url_for(image)) %>
         <%= cl_image_tag(image.key, height: 200, width: 200) %>
         <p style="text-align: center;">
-          <%= button_to 'Delete Image', delete_car_image_car_path(@car),params: {image_id:image.id}, method: :delete, data: { confirm: 'Are you sure?' } %>
+          <%= button_to 'Delete Image', delete_car_image_car_path(@car),params: {image_id:image.id}, method: :delete, data: { confirm: 'Are you sure?' }%>
```

Comment:
let's not make changes to existing functionality if not required, revert this change.